mysql-test/suite/encryption/r/innodb_default_key.result

```diff
@@ -93,5 +93,26 @@ include/assert.inc [Table t7 should be included in INNODB_TABLESPACES_ENCRYPTION
 drop table t1,t2,t3,t4,t5,t6,t7,t8,t9;
 SET GLOBAL default_table_encryption = ONLINE_FROM_KEYRING_TO_UNENCRYPTED;
 # Wait for all tables to get decrypted encrypted
-SET GLOBAL default_table_encryption = OFF;
-SET GLOBAL innodb_encryption_threads = 0;
+# restart:--default-table-encryption=ONLINE_TO_KEYRING --innodb-encryption-rotate-key-age=15 --innodb-encryption-threads=4 --innodb-default-encryption-key-id=4294967295 --log-error=MYSQL_TMP_DIR/my_restart.err
+# restart:--default-table-encryption=ONLINE_TO_KEYRING --innodb-encryption-rotate-key-age=15 --innodb-encryption-threads=4 --innodb-default-encryption-key-id=-1 --log-error=MYSQL_TMP_DIR/my_restart.err
+SET innodb_default_encryption_key_id=4294967295;
+ERROR 42000: Variable 'innodb_default_encryption_key_id' can't be set to the value of '4294967295'
+SET innodb_default_encryption_key_id=-1;
+ERROR 42000: Variable 'innodb_default_encryption_key_id' can't be set to the value of '-1'
+SET innodb_default_encryption_key_id=0;
+CREATE TABLE t_exceeded_id (a varchar(255)) ENCRYPTION_KEY_ID=4294967295;
+ERROR HY000: InnoDB system key id cannot have value greater than 4294967294
+CREATE TABLE t_below_0 (a varchar(255)) ENCRYPTION_KEY_ID=-1;
+ERROR 42000: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-1' at line 1
+CREATE TABLE t_MAX (a varchar(255)) ENCRYPTION_KEY_ID=4294967294;
+CREATE TABLE t_MIN (a varchar(255)) ENCRYPTION_KEY_ID=0;
+CREATE TABLESPACE ts_exceeded_id ENCRYPTION_KEY_ID=4294967295;
+ERROR HY000: InnoDB system key id cannot have value greater than 4294967294
+CREATE TABLESPACE ts_below_0 ENCRYPTION_KEY_ID=-1;
+ERROR 42000: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-1' at line 1
```

Comment:
I think you should move these newly added test scenarios to sys_vars.test for innodb-default-encryption-key-id

Reply:
moved